python/dllib/src/bigdl/dllib/utils/utils.py

```diff
@@ -212,7 +216,7 @@ def convert_for_cols(row, cols):
             elif isinstance(row[name], DenseVector):
                 result.append(row[name].values.astype(np.float32))
             else:
-                invalidInputError(isinstance(row[name], SparseVector),
+                invalidInputError(not isinstance(row[name], SparseVector),
```

Comment:
Maybe we shouldn't add `not` here...

Reply:
This is a comment sent by @cyita

Reply:
It seems to be irrelevant to the general purpose of this PR.  We could raise a new PR to fix this (if needed). 

Besides, to fix this, we may need to add some extra logic for converting a `SparseVector` to numpy ndarray, and then raise error for other input cases.

Reply:
sure, I will split this for another PR.